SalesforceCommerceCloud/pwa-kit
# Logging cid from res header isntead of req in local development

# Description
We currently log the http request using morgan (express middleware) for dev server, we want to move that to remote server so we can have the logs for both. This PR also makes sure the the correlation id will be logged via res.locals.requestId



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR
- Pull code, rebuild packages
-  Run Retail App
- Confirm that CID is available for SCAPI requests. This is just an internal ID as we haven't integrated our CID with Retail App yet, but it is suffice to identify the requests in the logging systems 

![image](https://user-images.githubusercontent.com/52219283/200935007-4a6f8a63-3525-40be-a2d6-9e7e9e661ed2.png)

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
